VSCWorkspace/Bakery/bakeryHome.html

```diff
@@ -19,7 +19,7 @@
               <ul id = "sub1" class = "subMenu" onmouseleave = "closeSubnav(this)">
                 <li><a id= "catSub" href="#" onmouseover = "openSubnav(sub2)">Tortas</a>
                     <ul onmouseleave = "closeSubnav(this)" class = "subMenu" id = "sub2">
-                        <li><a href="#">Tradicionales</a></li>
+                        <li><a href="/tortasTradicionales.html" target="_blank">Tradicionales</a></li>
```

Comment:
It seems like target="_blank" should be target="_self"

Opening the link in a new tab seems messy, especially if the user wants to see several items.  Right now, there isn't a way to link back to the main page, so the experience just dies after clicking tortas tradicionales, but that can be added later.

Reply:
You're right, my friend. I was actually thinking about opening that page in the main page. I'll change it.